admin/views/school/pupils-not-taking-check.ejs

```diff
@@ -0,0 +1,40 @@
+<div class="grid-row">
+    <div class="column-one-half">
+        <%- partial('partials/_breadcrumbs', { breadcrumbs: breadcrumbs }) %>
+    </div>
+    <div class="column-one-half">
+        <%- partial('partials/_name_and_sign_out') %>
+    </div>
+</div>
+
+<div class="grid-row" aria-describedby="heading-xlarge">
+    <div class="column-two-thirds" id="checkWindowsForm">
+        <h1 class="heading-xlarge" id="heading-xlarge" tabindex="0">Pupils not taking the check</h1>
+
+        <p class="lede">Click on the button below to add reasons for pupils who are not taking the check</p>
+
+        <p>
+            <a class="button button-primary" href="#">Add reason for pupil</a>
+        </p>
+
+        <br />
```

Comment:
Can we replace brs with css to match the expected behaviour?

Reply:
@GeorgeChatzigiannis I really don't want another CSS class just to add space once ever.